### PR TITLE
Move ASSERT() into test setup function

### PR DESCRIPTION
**Pull request by j-petit**
### Description
As discussed in [this PR review](https://github.com/ros-planning/moveit/pull/1657#discussion_r348925925), instead of an extra test case the robot setup should be tested in each run.

### CODE REVIEW

File: moveit_core/planning_scene/test/test_multi_threaded.cpp [hunk 1]
```diff
@@ -76,6 +76,8 @@ class CollisionDetectorThreadedTest : public testing::Test
     robot_model_ok_ = static_cast<bool>(robot_model_);
```
Comment (rhaschke) [inline moveit_core/planning_scene/test/test_multi_threaded.cpp]: Please replace this line with your assertion.
The following statements actually already rely on a valid robot_model_.

Comment (j-petit) [reply moveit_core/planning_scene/test/test_multi_threaded.cpp]: You're right, changed it.

**Comment by rhaschke** [comment]
@j-petit Could you address Travis issues, please.

**Comment by j-petit** [comment]
Why does this fail for kinetic but not for melodic?

**Comment by j-petit** [comment]
Thanks @rhaschke

**Pull request merged**